src/visitors/defuse_analyze_visitor.cpp

```diff
@@ -270,6 +272,73 @@ void DefUseAnalyzeVisitor::visit_verbatim(ast::Verbatim& node) {
     }
 }
 
+/// unsupported statements : we aren't sure how to handle this "yet" and
+/// hence variables used in any of the below statements are handled separately
+
+void DefUseAnalyzeVisitor::visit_reaction_statement(ast::ReactionStatement& node) {
+    visit_unsupported_node(node);
+}
+
+void DefUseAnalyzeVisitor::visit_non_lin_equation(ast::NonLinEquation& node) {
+    visit_unsupported_node(node);
+}
+
+void DefUseAnalyzeVisitor::visit_lin_equation(ast::LinEquation& node) {
+    visit_unsupported_node(node);
+}
+
+void DefUseAnalyzeVisitor::visit_partial_boundary(ast::PartialBoundary& node) {
+    visit_unsupported_node(node);
+}
+
```

Comment:
See my comments for `perf_visitor.hpp` which are applicable here. Some files I reviewed from the bottom :)